### PR TITLE
Added horse jump event and the event-entity event value for it

### PR DESCRIPTION
**Pull request by DeltaRays**
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

File: .idea/jarRepositories.xml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (APickledWalrus) [inline .idea/jarRepositories.xml:1]: Can you get rid of this file? Other than this, looks good!

Comment (DeltaRays) [reply .idea/jarRepositories.xml:1]: Oops yeah

Comment (DeltaRays) [reply .idea/jarRepositories.xml:1]: I'll finish working on fishing and I'll do that

**Review by APickledWalrus — APPROVED** [review]
Looks good. 

It might be a good idea to add an expression to get the power of this jump though. Maybe syntax like `[the] [jump] power`

**Comment by DeltaRays** [comment]
Yeah, unluckily it was transferred to be client side some versions ago

**Comment by APickledWalrus** [comment]
> 
> 
> Yeah, unluckily it was transferred to client side some versions ago

I think you can still get the power though, just not set. Thinking about it more, it probably wouldn't be super useful actually. If someone *really* wanted it they could probably use mirror or something.

**Comment by DeltaRays** [comment]
I'll look into it

**Comment by DeltaRays** [comment]
Yeah setting the power was deprecated back in 1.9, so it's probably not that useful

**Comment by APickledWalrus** [comment]
This PR should be good to go then

**Review by Wealthyturtle — APPROVED** [review]

**Review by ShaneBeee — APPROVED** [review]
Looks good to me

**Pull request merged**